### PR TITLE
Blog post ux

### PR DESCRIPTION
**Pull request by matthiasdebaat**
## Proposed change

This PR creates a new UX design blog post. My goal is to build a UX community and inform our users. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @matthiasdebaat, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19747).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_posts/2021-10-12-user-experiences-and-designers.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "User Experiences and Designers"
+description: "..."
```
Comment (bramkragten) [inline source/_posts/2021-10-12-user-experiences-and-designers.markdown]: You should add a description :-)

File: source/_posts/2021-10-12-user-experiences-and-designers.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Designers
+Last year alone, there were over 8,000 GitHub contributors to Home Assistant. That's really fantastic! With a Designer-to-Developer ratio of 1 per 8.000, it’s easy to get lost in the backend of design and forget the users. This can result in inconsistent user interfaces or designs that could be more user-friendly. For now we’re working on three areas.
+
+### UX contributors
+We want to make it as easy for designers to contribute as it is for developers. There’s a lot a designer can contribute to. For example making it easier to work with light groups or getting started with automating your house. If you have a design you really think we should look at? Would you like to help build our own Design System? Do you have experience with user testing? Do you have strategic design ideas? Want to organize an online design workshop? Or just want to meet fellow designers? Meet us at [devs_ux Discord channel](https://discord.gg/BPBc8rZ9) or join us on [GitHub discussions](https://github.com/home-assistant/frontend/discussions/10147).
+
```
Comment (frenck) [inline source/_posts/2021-10-12-user-experiences-and-designers.markdown]:
```suggestion
We want to make it as easy for designers to contribute as it is for developers. There’s a lot a designer can contribute to. For example, making it easier to work with light groups or getting started with automating your house. If you have a design you really think we should look at? Would you like to help build our own Design System? Do you have experience with user testing? Do you have strategic design ideas? Want to organize an online design workshop? Or just want to meet fellow designers? Meet us at [devs_ux Discord channel](https://discord.gg/BPBc8rZ9) or join us on [GitHub discussions](https://github.com/home-assistant/frontend/discussions/10147).

```

File: source/_posts/2021-10-12-user-experiences-and-designers.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+### Tooling
+We want it to be easy to start creating and iterating on design mockups for Home Assistant. We want anyone that is interested to be able to jump in. To achieve this we’ve been hard at work at re-creating the components that make up the Home Assistant interface as a [DesignKit for Figma](https://www.figma.com/community/file/967153512097289521/Home-Assistant-DesignKit). This allows anyone to experiment with the Home Assistant interface and quickly try out different concepts. Try it yourself and feel free to share your designs.
+
+### Documentation
+With clear guidelines you can ensure speed and quality. That’s why we’re working on design documentation that will be available soon. It’s goal is to help designers and developers make better choices. Got suggestions or want to help with our guidelines? Meet us at [devs_ux Discord channel](https://discord.gg/BPBc8rZ9).
```
Comment (frenck) [inline source/_posts/2021-10-12-user-experiences-and-designers.markdown]:
```suggestion
With clear guidelines, you can ensure speed and quality. That’s why we’re working on design documentation that will be available soon. Its goal is to help designers and developers make better choices. Got suggestions or want to help with our guidelines? Meet us at [devs_ux Discord channel](https://discord.gg/BPBc8rZ9).
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**